multimap/src/test/java/org/infinispan/multimap/impl/DistributedMultimapPairCacheTest.java

```diff
@@ -138,4 +145,22 @@ public void testContainsProperty() {
             })
             .thenAccept(b -> assertThat(b).isFalse())));
    }
+
+   public void testSubSelect() {
+      assertFromAllCaches(multimap -> await(multimap.set("sub-select-test", Map.entry(toBytes("oihana"), OIHANA), Map.entry(toBytes("koldo"), KOLDO))
+            .thenCompose(ignore -> multimap.subSelect("sub-select-test", 1))
+            .thenAccept(m ->
+                  assertThat(convertKeys(m))
+                        .hasSize(1)
+                        .containsAnyOf(Map.entry("oihana", OIHANA), Map.entry("koldo", KOLDO)))));
+   }
+
+   public void testGetMultiple() {
```

Comment:
Should we add a test for what it returns when the key isn't present? Should we return an entry with a null key or just not return it?